README.md

```diff
@@ -8,8 +8,14 @@ FFmpegInteropX is an open-source project that aims to provide an easy way to use
 
 FFmpegInteropX is a much **improved fork** of the original [Microsoft project](git://github.com/Microsoft/FFmpegInterop).
 
-**Some of the important improvements:**
+#### Latest Releases:
+- FFmpegInteropX: 0.9.0
+  - First official NuGet Packet!
```

Comment:
(admittedly pedantic point)

I believe NuGet artifacts are commonly referred to as "packages" rather than "packets".